packages/block-library/src/heading/autogenerate-anchors.js

```diff
@@ -125,7 +125,8 @@ export default function useGeneratedAnchor( clientId, anchor, content ) {
 		},
 		[ clientId ]
 	);
-	if ( isNil( anchor ) || startsWith( anchor, 'wp-' ) ) {
+	// eslint-disable-next-line eqeqeq
+	if ( null == anchor || 0 === anchor.indexOf( 'wp-' ) ) {
```

Comment:
I meant the same native fn: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith :)

Reply:
No need for that now, that line was just deleted :laughing: